batch/sql/add-user-resources-ready-cores-token.py

```diff
@@ -0,0 +1,18 @@
+import asyncio
+from gear import Database
+
+
+async def main():
+    db = Database()
+    await db.async_init()
+
+    await db.just_execute('CALL insert_ready_cores_tokens();')
+
+    users = db.execute_and_fetchall('SELECT user FROM user_resources;')
+    async for user in users:
+        await db.just_execute('CALL insert_user_resources_tokens(%s)',
+                              (user,))
+
+
+loop = asyncio.get_event_loop()
+loop.run_until_complete(main())
```

Comment:
To do this in SQL we'd need to create a procedure that does this, run it, then drop it, and you're thinking is the python is simpler, yeah?

Reply:
The ready cores transformation could be done in SQL, but doing the one for the users seemed easier to just iterate in Python.